### PR TITLE
Formatting markdown files

### PR DESCRIPTION
**Pull request by DennisDyallo**
# Description

Formatting md-files, applying proper format of tables,
as well as adding them to the solution

### CODE REVIEW

File: Yubico.YubiKey/docs/users-manual/application-fido2/apdu/get-info.md [hunk 1]
```diff
@@ -25,8 +25,9 @@ CLA | INS | P1 | P2 | Lc | Data | Le
 Total Length: *variable + 2*\
 Data Length: *variable*
 
-Data | SW1 | SW2
-:---: | :---: | :---:
+      Data      | SW1 | SW2
+
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-fido2/apdu/get-info.md:29]: extra line causing problems

File: Yubico.YubiKey/docs/users-manual/application-fido2/apdu/get-key-agree.md [hunk 2]
```diff
@@ -44,8 +44,9 @@ the following:
 Total Length: *variable + 2*\
 Data Length: *variable*
 
-Data | SW1 | SW2
-:---: | :---: | :---:
+      Data      | SW1 | SW2
+
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-fido2/apdu/get-key-agree.md:48]: extra line causing problems

File: Yubico.YubiKey/docs/users-manual/application-fido2/apdu/enum-rps-begin.md
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-fido2/apdu/enum-rps-begin.md:1]: File not in toc.

File: Yubico.YubiKey/docs/users-manual/application-fido2/apdu/enum-rps-next.md
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-fido2/apdu/enum-rps-next.md:1]: File not in toc.

File: Yubico.YubiKey/docs/users-manual/application-fido2/fido2-reset.md
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-fido2/fido2-reset.md:1]: File not in toc.

File: Yubico.YubiKey/docs/users-manual/application-otp/how-to-program-a-challenge-response-credential.md [hunk 3]
```diff
@@ -18,45 +18,78 @@ limitations under the License. -->
...
-To enable this setting, add the [UseButton()](xref:Yubico.YubiKey.Otp.Operations.ConfigureChallengeResponse.UseButton(System.Boolean)) method to your ``ConfigureChallengeResponse()`` operation.
+To enable this setting, add
+the [UseButton()](xref:Yubico.YubiKey.Otp.Operations.ConfigureChallengeResponse.UseButton(System.Boolean)) method to
+your ``ConfigureChallengeResponse()`` operation.
 
 > [!NOTE]
-> If a YubiKey has been configured to require a button touch, you must make sure to alert the user of this requirement during a challenge-response operation. This can be accomplished by calling the [UseTouchNotifier()](xref:Yubico.YubiKey.Otp.Operations.CalculateChallengeResponse.UseTouchNotifier(System.Action)) method when sending a challenge to a YubiKey via ``CalculateChallengeResponse()``.
+> If a YubiKey has been configured to require a button touch, you must make sure to alert the user of this requirement
+> during a challenge-response operation. This can be accomplished by calling
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-otp/how-to-program-a-challenge-response-credential.md:85]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-otp/how-to-program-a-static-password.md [hunk 4]
```diff
@@ -142,24 +183,36 @@ using (OtpSession otp = new OtpSession(yubiKey))
...
+holding the contact of the YubiKey for 8-15 seconds. When the contact is released, the indicator light will flash.
+Touching the contact again confirms the change, and the new static password is generated and stored in the OTP slot.
 
 To enable the manual update feature, you must:
 
 1. set the [static ticket flag](xref:Yubico.YubiKey.Otp.ConfigurationFlags.StaticTicket), then
 
-1. set [AllowManualUpdate()](xref:Yubico.YubiKey.Otp.Operations.ConfigureStaticPassword.AllowManualUpdate%28System.Boolean%29) to ``true`` when calling ``ConfigureStaticPassword()``.
+1.
+
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-otp/how-to-program-a-static-password.md:198]: Extra line causing issues.

File: Yubico.YubiKey/docs/users-manual/application-piv/apdu/put-vendor-data.md
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-piv/apdu/put-vendor-data.md:1]: File not in toc

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/add-credential.md [hunk 5]
```diff
@@ -43,31 +50,37 @@ The rest of the input data is related to the new credential:
 Further information is found in the [Data](#data) section.
 
 > [!NOTE]
-> The label may only contain characters that can be encoded with UTF-8, and its UTF-8 byte count must be between 1 and 64. Non-printing characters are allowed, as long as they can be encoded with UTF-8. For example, null (UTF-8: 0x00) and Right-To-Left Mark U+200F (UTF-8: 0xE2 0x80 0x8F) would be accepted. Since the label is used for display purposes, it is recommended to prefer printable characters.
+> The label may only contain characters that can be encoded with UTF-8, and its UTF-8 byte count must be between 1 and
+
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/add-credential.md:54]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/add-credential.md [hunk 6]
```diff
@@ -21,7 +21,10 @@ Store long-lived keys in the YubiHSM Auth application by creating a new [credent
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/add-credential.md:25]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/change-management-key.md [hunk 7]
```diff
@@ -18,15 +18,23 @@ limitations under the License. -->
...
+the application, see [ResetApplicationCommand](xref:YubiHsmAuthCmdResetApplication). Supplying the correct management
+key before the management key is blocked will reset the retry counter to 8.
 
 ## Available
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/change-management-key.md:35]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/delete-credential.md [hunk 8]
```diff
@@ -24,7 +24,10 @@ Remove a credential from the YubiHSM Auth application.
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/delete-credential.md:28]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/get-aes128-session-keys.md [hunk 9]
```diff
@@ -24,7 +24,10 @@ Get the SCP03 session keys from an AES-128 credential.
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/get-aes128-session-keys.md:28]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/get-application-version.md [hunk 10]
```diff
@@ -24,7 +24,10 @@ Get the version of the YubiHSM Auth application returned as a major, minor, and
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/get-application-version.md:28]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/get-management-key-retries.md [hunk 11]
```diff
@@ -20,13 +20,20 @@ limitations under the License. -->
...
+application, see [ResetApplicationCommand](xref:YubiHsmAuthCmdResetApplication). Supplying the correct management key
+before the management key is blocked will reset the retry counter to 8.
 
 ## Available
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/get-management-key-retries.md:34]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/list-credentials.md [hunk 12]
```diff
@@ -18,13 +18,17 @@ limitations under the License. -->
...
+Get the public properties of all credentials present in the YubiHSM Auth application along with the number of retries
+remaining for each.
 
 ## Available
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/list-credentials.md:29]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/reset-application.md [hunk 13]
```diff
@@ -18,13 +18,17 @@ limitations under the License. -->
...
+Reset the YubiHSM Auth application. All credentials will be deleted, the management key will be reset to the default
+value (all zeros), and the management key retry counter will be reset to 8.
 
 ## Available
 
 All YubiKeys with the YubiHSM Auth application (included in firmware version 5.4.3 and later).
 > [!NOTE]
-> Use the .NET API's [HasFeature()](xref:Yubico.YubiKey.YubiKeyFeatureExtensions.HasFeature%28Yubico.YubiKey.IYubiKeyDevice%2CYubico.YubiKey.YubiKeyFeature%29) method to check if a key has the YubiHSM Auth application.
+> Use the .NET
+>
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/commands/reset-application.md:29]: Extra line causing formatting issues.

File: Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/yubihsm-auth-session.md [hunk 14]
```diff
@@ -38,19 +44,32 @@ using (var YubiHsmAuthSession = new YubiHsmAuthSession(yubiKeyToUse))
...
+Calculate session keys from an AES-128 credential. These session keys are used to establish a secure session with a
+YubiHSM 2 device. | n/a |
+| [Get application version](xref:Yubico.YubiKey.YubiHsmAuth.YubiHsmAuthSession.GetApplicationVersion) | Get the version
+of the YubiHSM Auth application returned as a major, minor, and patch value. | n/a |
+| [Get management key retries](xref:Yubico.YubiKey.YubiHsmAuth.YubiHsmAuthSession.GetManagementKeyRetries) | Get the
+number of retries remaining for the management key. | n/a |
+| [List credentials](xref:Yubico.YubiKey.YubiHsmAuth.YubiHsmAuthSession.ListCredentials) | Get the public properties of
+all credentials in the YubiHSM Auth application, along with the number of retries remaining for each. | n/a |
+| [Reset application](xref:Yubico.YubiKey.YubiHsmAuth.YubiHsmAuthSession.ResetApplication) | Reset the YubiHSM Auth
+application, which will delete all credentials and set the management key to its default value (all zeros). | n/a |
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/yubihsm-auth-session.md:75]: Table not rendering correctly.

Comment (DennisDyallo) [reply Yubico.YubiKey/docs/users-manual/application-yubihsm-auth/yubihsm-auth-session.md]: Addressed all of these now hopefully!

**Pull request merged**